src/attach-shadow.js

```diff
@@ -73,10 +73,11 @@ export class ShadyRoot {
     this._slotList = null;
     this._slotMap = null;
     this._pendingSlots = null;
+    this._initialChildren = null;
     // fast path initial render: remove existing physical dom.
-    for (let i=0, l=c$.length; i < l; i++) {
-      removeChild.call(host, c$[i])
-    }
+    recordChildNodes(host, childNodes(host));
```

Comment:
Why record child nodes if the element is `_createdWhileLoading`?  Avoids the potential of double linking/patching work, since it's not guaranteed to be correct anyway?